lib/android/app/src/main/java/com/wix/reactnativenotifications/RNNotificationsModule.java

```diff
@@ -89,17 +90,14 @@ public void getInitialNotification(final Promise promise) {
             }
 
             result = Arguments.fromBundle(notification.asBundle());
```

Comment:
why did we remove it?

Reply:
App navigates to notification payload again taking it from intent, this has been tested